source/theme/styles/components/_print.scss

```
@@ -0,0 +1,57 @@
+@media print {
+
+  body {
+    padding: 1.5em 1em;
```

I see a mix of px & em. Can we use pixels?

---

Sure how many pixels should I pick, just something that looks good?

---

@willemstuursma you can convert em to pixels.

```
1 em   = 16px
1.5em = 24px
```

```suggestion
    padding: 24px 16px;
```

---

@willemstuursma you can convert em to pixels.

```
1 em   = 16px
1.5em = 24px
```

```suggestion
    padding: 24px 16px;
```